### DISCUSSION TITLE
Use and document Trivy JSON report schema version 2 in the Trivy documentation

### DISCUSSION THREAD

**Discussion by fniessink** [aquasecurity/trivy#7552] [Documentation]
### Description

When generating a Trivy report with JSON format (for example `trivy image --debug --offline-scan --input target/tar_images/example_image.tar -f json -o target/testresults/trivy/trivy-results.json`), it uses schema version 2 as discussed [here](https://github.com/aquasecurity/trivy/discussions/1050). However, the documentation shows an [example](https://aquasecurity.github.io/trivy/v0.55/docs/configuration/reporting/#json) conform schema version 1 and there's no mention of a schema.

### Link

https://aquasecurity.github.io/trivy/v0.55/docs/configuration/reporting/#json

### Suggestions

1. Update the example JSON report to conform to schema version 2.
2. Document schema version 2 as part of the documentation.

**Comment by knqyf263** [comment]
Both suggestions make sense to me. Created https://github.com/aquasecurity/trivy/issues/7553. Thanks.

**Reply by mtausig** [reply]
The changes made in PR #8188 only fix the first suggestion from the OP (remove references to the old schema) but do not tackle the second one (document the actual schema).
